documentation/src/main/asciidoc/topics/proc_running_embedded.adoc

```diff
@@ -0,0 +1,54 @@
+[id='run_library']
+= Running {brandname} as an Embedded Library
+Learn how to run {brandname} as an embedded data store in your project.
+
+.Procedure
+
+* Initialize the default Cache Manager and add a cache definition as follows:
+
+[source,java,options="nowrap"]
+----
+import org.infinispan.Cache;
+import org.infinispan.commons.api.CacheContainerAdmin;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.manager.DefaultCacheManager;
+
+...
+
+      GlobalConfigurationBuilder global = GlobalConfigurationBuilder.defaultClusteredBuilder();
+            DefaultCacheManager cacheManager = new DefaultCacheManager(global.build());
+            ConfigurationBuilder builder = new ConfigurationBuilder();
+            builder.clustering().cacheMode(CacheMode.DIST_SYNC);
+      cacheManager.administration().getOrCreate("myCache", builder.build());
+----
+
+The preceding code initializes a default, local Cache Manager. Cache Managers
```

Comment:
Incorrect: the above code creates a clustered cache manager.

Reply:
argh. sorry that should be totally obvious. I guess I got hold of a bad batch of TP moonshine. fixed.